src/components/phase-banner/phase-banner.njk

```diff
@@ -1,7 +1,13 @@
 {% from "phase-banner/macro.njk" import govukPhaseBanner %}
 {{ govukPhaseBanner(classes='',
   {
-    "tag": "alpha",
-    "message": "This part of GOV.UK Frontend is being built"
+    "tag":
+    {
+      "text": "alpha",
+      "html": "alpha",
+      "classes": "doing-something"
+    },
+    "text": "This part of GOV.UK Frontend is being built",
+    "html" : "This is a new service – your <a href='#''>feedback</a> will help us to improve it."
```

Comment:
It only makes sense to pass either `text` or `html`, but not both - if `html` is provided then `text` is ignored. Suggest using just the `html` argument for now.